src/app/shared/components/autocomplete/autocomplete.component.scss

```diff
@@ -0,0 +1,3 @@
+input::-webkit-calendar-picker-indicator {
```

Comment:
i would assume this style will be used for every auto complete components so it is better to have in the global.scss assuming the styling is not given by the design system we have?
This would be some portable style should we had to change js framework.